test/e2e/adminapi.go

```diff
@@ -13,14 +13,22 @@ import (
 	"net/http"
 	"net/url"
 	"os"
+
+	"github.com/Azure/ARO-RP/pkg/api/admin"
 )
 
 func adminRequest(ctx context.Context, method, path string, params url.Values, in, out interface{}) (*http.Response, error) {
 	if os.Getenv("RP_MODE") != "development" {
 		return nil, errors.New("only development RP mode is supported")
 	}
 
-	adminAPIBaseURI := "https://localhost:8443/admin"
+	if params == nil {
+		params = url.Values{}
+	}
+
+	params.Add("api-version", admin.APIVersion)
```

Comment:
note that this will add api-version on some REST calls that don't need it

Reply:
I think that's fine. We just ignore it if we don't need it. I think I'll keep it for simplicity.